### DISCUSSION TITLE
现在除了一线以及新一线的核心房产，其他地方的房产应该都不能买了吧

### DISCUSSION THREAD

**Discussion by rechardlc** [WeNeedHome/SummaryOfLoanSuspension#105] [General]
考虑买房~头疼~

**Comment by xujiyou** [comment]
郑州不是新一线吗

**Reply by xuexuegege** [reply]
> 郑州不是新一线吗

按理说新一线是沿海工商业中心比如杭州南京苏州和内地跨区辐射中心城市如重庆成都武汉

**Reply by zhaokai5520** [reply]
郑州不算吗

**Reply by lynn1885** [reply]
> 

武汉烂尾也已经第二多了

**Reply by alexwjj** [reply]
郑州除了人多，没有任何优势

**Reply by woyewangle** [reply]
来南方沿海发展吧，治安又好 烂尾又少 黑社会也没那么明目张胆 也没有赋码爷

**Comment by SystemCodes** [comment]
刚需你该上还是得上，你肉身在中国。你就得为房子一辈子打工。
通过小小土地控制13亿多被统治阶级一辈子，设计是非常好的。

**Reply by gt562559842** [reply]
其实了解历史就能知道，赋税是每个朝代必须的，只是看形式是怎样，没有国家就没有个体，这其实不能怪国家；要怪就怪那些黑心的开发商和不遵守规则的银行

**Reply by ChrisJonasou** [reply]
不少口子当初就是国家“立法”的时候埋进去的。比如村镇银行，当时很多官员和学者反对，说准入门槛太低，有风险，结果还是强行通过。

银行贷款也是，当初学的国外，让银行给期房贷款。本来是个很好的资金流转的手段，但是国外对银行有立法限制，如果期房没交，买房者一分贷款不用付，银行自行承担，在这种制度下银行必定对开发商严加约束，不然银行自己就没了。我们国家根本不管这个，把所有责任全部推给买房者。制度设计的时候就有问题，故意埋了钉子。

今年这个时间点爆出来这么多事情，我很怀疑和董事会有关，不知道是不是十变十五，有人斗法，故意搞一堆事情。

各位这两年多加小心。

**Reply by AntEffect** [reply]
> 不少口子当初就是国家“立法”的时候埋进去的。比如村镇银行，当时很多官员和学者反对，说准入门槛太低，有风险，结果还是强行通过。
> 
> 银行贷款也是，当初学的国外，让银行给期房贷款。本来是个很好的资金流转的手段，但是国外对银行有立法限制，如果期房没交，买房者一分贷款不用付，银行自行承担，在这种制度下银行必定对开发商严加约束，不然银行自己就没了。我们国家根本不管这个，把所有责任全部推给买房者。制度设计的时候就有问题，故意埋了钉子。
> 
> 今年这个时间点爆出来这么多事情，我很怀疑和董事会有关，不知道是不是十变十五，有人斗法，故意搞一堆事情。
> 
> 各位这两年多加小心。

听说村镇银行是当时的几个头头为了方便捞钱故意开的口子

**Reply by wengxianxun** [reply]
> 不少口子当初就是国家“立法”的时候埋进去的。比如村镇银行，当时很多官员和学者反对，说准入门槛太低，有风险，结果还是强行通过。
> 
> 银行贷款也是，当初学的国外，让银行给期房贷款。本来是个很好的资金流转的手段，但是国外对银行有立法限制，如果期房没交，买房者一分贷款不用付，银行自行承担，在这种制度下银行必定对开发商严加约束，不然银行自己就没了。我们国家根本不管这个，把所有责任全部推给买房者。制度设计的时候就有问题，故意埋了钉子。
> 
> 今年这个时间点爆出来这么多事情，我很怀疑和董事会有关，不知道是不是十变十五，有人斗法，故意搞一堆事情。
> 
> 各位这两年多加小心。

中秋大选完是不是要开始秋后问斩了

**Reply by AntEffect** [reply]
> > 不少口子当初就是国家“立法”的时候埋进去的。比如村镇银行，当时很多官员和学者反对，说准入门槛太低，有风险，结果还是强行通过。
> > 
> > 银行贷款也是，当初学的国外，让银行给期房贷款。本来是个很好的资金流转的手段，但是国外对银行有立法限制，如果期房没交，买房者一分贷款不用付，银行自行承担，在这种制度下银行必定对开发商严加约束，不然银行自己就没了。我们国家根本不管这个，把所有责任全部推给买房者。制度设计的时候就有问题，故意埋了钉子。
> > 
> > 今年这个时间点爆出来这么多事情，我很怀疑和董事会有关，不知道是不是十变十五，有人斗法，故意搞一堆事情。
> > 
> > 各位这两年多加小心。
> 
> 中秋大选完是不是要开始秋后问斩了

想多了，说不一定就是自己的马仔

**Reply by Gu0Hao** [reply]
十变十五？悬了。

**Reply by last77** [reply]
> 

对 一切都是太监的错

**Reply by qq912276337** [reply]
刚需？？？你怎么定义刚需？？？吃饭活下去才是刚需，可用可不用暂时可不用那不叫刚需，不要强行刚需

**Reply by seek-x2y** [reply]
> 不少口子当初就是国家“立法”的时候埋进去的。比如村镇银行，当时很多官员和学者反对，说准入门槛太低，有风险，结果还是强行通过。
> 
> 银行贷款也是，当初学的国外，让银行给期房贷款。本来是个很好的资金流转的手段，但是国外对银行有立法限制，如果期房没交，买房者一分贷款不用付，银行自行承担，在这种制度下银行必定对开发商严加约束，不然银行自己就没了。我们国家根本不管这个，把所有责任全部推给买房者。制度设计的时候就有问题，故意埋了钉子。
> 
> 今年这个时间点爆出来这么多事情，我很怀疑和董事会有关，不知道是不是十变十五，有人斗法，故意搞一堆事情。
> 
> 各位这两年多加小心。

当初村镇银行的反对声音在哪里？有什么资料可查吗？

**Comment by lynn1885** [comment]
看热力图, 和经济情况差不多, 东北和广义上的中部地区在塌方

**Comment by woshizengling** [comment]
准备在长沙买房的我，看着这一个个烂尾楼盘，瑟瑟发抖

**Reply by gu-fan** [reply]
没买抖啥，偷着乐还来不及

**Reply by woyewangle** [reply]
下半都要剧震，你现在还想着上车？ 真的是大侠。侠之大者 为国接盘

**Reply by MatthewDevelop** [reply]
只买现房! 当然,你不买我不买,明天还能降五百!

**Reply by woyewangle** [reply]
你不卖我不卖，眼泪掉下来。
你不生我不生，高房价胎死腹中

**Comment by cm-liang** [comment]
上海应该算一线吧

**Comment by cccp057** [comment]
你买现房或者二手房不就完了，肯定不用担心烂尾，建成几年的房子跟新的没什么区别

**Reply by Jackwew** [reply]
贵啊，

**Comment by rechardlc** [comment]
哎~太难了~啥时候到头啊~

**Comment by Gie96** [comment]
就这么说吧，除非买现房不然中国的房产都别买了。买现房尽可能也别背太久的房贷，经济很差不好说以后怎么样
